components/hab/static/template_gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+results/
```

Comment:
This also might get weird with multi-package repositories (like `habitat` itself), where you would have multiple directories in a repository that you might run `hab pkg init` in. Should we add `/path/to/subproject/results` to a top-level `.gitignore`? Would we instead recommend that if you use that pattern you should always enter your studio from the top level directory and then run `build path/to/subproject` once you get in (so your artifacts always go into the top-level `results` directory?